msteams-platform/concepts/app-fundamentals-overview.md

```diff
@@ -23,7 +23,7 @@ In this section, learn how to:
 
 Teams as a platform offers you toolkits, libraries, and apps at every stage of app development. Let's break it down to app building lifecycle:
 
-:::image type="content" source="../assets/images/app-fundamentals/plan-app.png" alt-text="Illustration shows planning your app":::
+:::image type="content" source="../assets/images/app-fundamentals/plan-app.png" alt-text="The screenshot is an example Illustration that describes about planning your app.":::
```

Comment:
First, this image is needs revisiting about what it wants to portray + how it lays out the app dev lifecycle. CC @v-shalinirobinson and @surbhigupta12 .

Second (unrelated-to-above) feedback is about alt text:
* It is not a screenshot.
* Illustration casing is wrong.
* This is not an example of anything.
* The meaning conveyed by the alt-text before and after the edit isn't much different, so why the update? 

CC @v-habibaumme , @v-psambrani .

Reply:
changed the screenshot to image.